proto/info/info.proto

```diff
@@ -677,6 +677,7 @@ enum APICapability {
   API_CAPABILITY_BOOT_MODE = 7; // Support different boot modes for Edge Applications (VMs)
   API_CAPABILITY_MTU = 8; // Allows to set MTU for network adapters and network instances
   API_CAPABILITY_ADAPTER_USER_LABELS = 9; // Supports user-defined shared network adapter labels
+  API_CAPABILITY_NTP_FQDN = 10; // Allow to set NTP server via FQDN instead of only IP
```

Comment:
clashes with https://github.com/lf-edge/eve-api/pull/74/files#diff-b6af1b4801076bb7a7c0b5f919ecab887a34b218f6a5f2bfdffa40845a5e618dR680

Reply:
I had to merge the other PR first due to high-priority. Please rebase and increase the enum field ID.

Reply:
Sorry, missed that this is already done :)